csm_web/frontend/src/components/MentorSection.js

```diff
@@ -587,7 +591,12 @@ function MentorSectionInfo({
         </InfoCard>
         <SectionSpacetime spacetime={spacetime} override={override}>
           {showModal === MentorSectionInfo.MODAL_STATES.SPACETIME_EDIT && (
-            <SpacetimeEditModal reloadSection={reloadSection} spacetimeId={spacetime.id} closeModal={closeModal} />
+            <SpacetimeEditModal
+              reloadSection={reloadSection}
+              spacetimeId={spacetime.id}
```

Comment:
No need to pass `spacetimeId` as a separate prop anymore if the whole `spacetime` is being passed in (this is just refactoring, what you have works).